src/test/resources/example.vcf.html

```diff
@@ -3,7 +3,7 @@
erson":{"affectedStatusValue":0,"maternalId":"0","affectedStatus":"MISSING","paternalId":"0","sex":"UNKNOWN_SEX","familyId":"MISSING_0","individualId":"NA00001","sexValue":0},"index":0},{"person":{"affectedStatusValue":0,"maternalId":"0","affectedStatus":"MISSING","paternalId":"0","sex":"UNKNOWN_SEX","familyId":"MISSING_1","individualId":"NA00002","sexValue":0},"index":1},{"person":{"affectedStatusValue":1,"maternalId":"0","affectedStatus":"UNAFFECTED","paternalId":"0","sex":"MALE","familyId":"FAM002","individualId":"James","sexValue":2},"index":-1},{"person":{"affectedStatusValue":2,"maternalId":"Jane","affectedStatus":"AFFECTED","paternalId":"Jimmy","sex":"MALE","familyId":"FAM001","individualId":"John","sexValue":2},"index":-1},{"person":{"affectedStatusValue":0,"maternalId":"0","affectedStatus":"MISSING","paternalId":"0","sex":"UNKNOWN_SEX","familyId":"MISSING_2","individualId":"NA00003","sexValue":0},"index":2},{"person":{"affectedStatusValue":1,"maternalId":"0","affectedStatus":"UNAFFECTED","paternalId":"0","sex":"FEMALE","familyId":"FAM001","individualId":"Jane","sexValue":1},"index":-1},{"person":{"affectedStatusValue":1,"maternalId":"0","affectedStatus":"UNAFFECTED","paternalId":"0","sex":"MALE","familyId":"FAM001","individualId":"Jimmy","sexValue":2},"index":-1}],"total":8},"phenotypes":{"items":[{"phenotypicFeaturesList":[{"type":{"id":"HP:123456","label":"HP:123456"}}],"phenotypicFeaturesCount":1,"subject":{"id":"Jimmy"}},{"phenotypicFeaturesList":[{"type":{"id":"test:Headache","label":"test:Headache"}}],"phenotypicFeaturesCount":1,"subject":{"id":"Unknown"}},{"phenotypicFeaturesList":[{"type":{"id":"OMIM:23456","label":"OMIM:23456"}}],"phenotypicFeaturesCount":1,"subject":{"id":"Jane"}}],"total":3},"records":{"items":[{"c":"20","p":14370,"i":["rs6054257"],"r":"G","a":["A"],"q":29.0,"n":{"H2":true,"DP":14,"NS":3,"AF":[0.5],"DB":true},"s":[{"gt":{"a":["G","G"],"p":true,"t":"hom_r"}},{"gt":{"a":["A","G"],"p":true,"t":"het"}},{"gt":{"a":["A","A"],"p":false,"t":"hom_a"}}]},{"c":"20","p":17330,"r":"T","a":["A"],"q":3.0,"f":["q10"],"n":{"DP":11,"NS":3,"AF":[0.017]},"s":[{"gt":{"a":["T","T"],"p":true,"t":"hom_r"}},{"gt":{"a":["T","A"],"p":true,"t":"het"}},{"gt":{"a":["T","T"],"p":false,"t":"hom_r"}}]},{"c":"20","p":1110696,"i":["rs6040355"],"r":"A","a":["G","T"],"q":67.0,"n":{"AA":"T","DP":10,"NS":2,"AF":[0.333,0.667],"DB":true},"s":[{"gt":{"a":["G","T"],"p":true,"t":"het"}},{"gt":{"a":["T","G"],"p":true,"t":"het"}},{"gt":{"a":["T","T"],"p":false,"t":"hom_a"}}]},{"c":"20","p":1230237,"r":"T","q":47.0,"n":{"AA":"T","DP":13,"NS":3},"s":[{"gt":{"a":["T","T"],"p":true,"t":"hom_r"}},{"gt":{"a":["T","T"],"p":true,"t":"hom_r"}},{"gt":{"a":["T","T"],"p":false,"t":"hom_r"}}]},{"c":"20","p":1234567,"i":["microsat1"],"r":"GTC","a":["G","GTCT"],"q":50.0,"n":{"AA":"G","DP":9,"NS":3},"s":[{"gt":{"a":["GTC","G"],"p":false,"t":"het"}},{"gt":{"a":["GTC","GTCT"],"p":false,"t":"het"}},{"gt":{"a":["G","G"],"p":false,"t":"hom_a"}}]}],"total":5}}}</script></head>
```

Comment:
suggestion: update the ApiIT to produce pretty printed output so that we can easily review changes